db/migrate/20180628102730_add_link_text_setting_to_page_translations.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddLinkTextSettingToPageTranslations < ActiveRecord::Migration
+  def change
+    reversible do |dir|
+      dir.up do
+        Page::Browse::RecordSets.add_translation_fields! settings_link_text: :string
```

Comment:
I think, this breaks saving of any other pages(Page, Page::Landing, Page::Error) through rails admin.
Are you sure a translation can be added to a setting?